src/solvers/smt2/smt2_conv.cpp

```diff
@@ -3940,10 +3942,14 @@ void smt2_convt::convert_rounding_mode_FPA(const exprt &expr)
     convert_expr(expr);
     out << ") roundTowardPositive ";
 
+    out << "(ite (= (_ bv3 " << width << ") ";
+    convert_expr(expr);
+    out << ") roundTowardZero ";
+
     // TODO: add some kind of error checking here
-    out << "roundTowardZero";
+    out << "roundTowardAway";
```

Comment:
Shouldn't this be `RoundNearestTiesToAway`?

Reply:
Indeed, fixed.